### PR TITLE
Add Design Guidelines to DVELP Cookbook

### PR DESCRIPTION
**Pull request by latimerjason**
- Updated Readme.md with link to design guidelines
- Add guidelines for formats, sizes, and best practice for designers working with DVELP.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ better.
   * [Ruby gems](/guides/Creating%20Ruby%20Gems%20bespoke%20for%20DVELP.md)
   * [Shopify](/guides/Shopify.md)
   * [Trello] (/guides/trello.md)
+  * [Design Guidelines] (/guides/design-guidelines.md)
```
Comment (tom-mullen) [inline Readme.md]: Alphabetical order please :-)

**Comment by latimerjason** [comment]
@tom-mullen - that change has been made. Can this be merged?

**Comment by latimerjason** [comment]
Any other feedback?

**Review by tom-mullen — APPROVED** [review]
🚢

**Pull request merged**